lib/cdo/slack.rb

```diff
@@ -19,6 +19,18 @@ class Slack
 
   SLACK_TOKEN = CDO.slack_token.freeze
 
+  # @param email [String] The email of the Slack user.
+  # @raise [ArgumentError] If the email does not correspond to a Slack user.
+  # @return [nil | String] The mention name for the Slack user.
+  def self.mention_name(email)
```

Comment:
I don't love this method name, for two reasons.

1. At a glance I thought we were using `mention` as a verb - as if calling this method would immediately `@mention` in slack somewhere.
2. Once I'd worked out that it just returns a string, I still expected it to return `"@brad"` including the leading `@`, when in reality it returns `"brad"`.

I wonder if one of `user_id`, `user_name`, or `user_name_from_email` would be a better option here.